### DISCUSSION TITLE
Table header staying white in dark mode

### DISCUSSION THREAD

**Discussion by nadeaumarc** [awcodes/filament-table-repeater#108] [Q&A] [answered]
Hello, I can't seem to find why my table repeater header stays white in dark mode. In light mode everything seems to be fine. 

Any idea what it could be?

![image](https://github.com/awcodes/filament-table-repeater/assets/127355317/923677f7-3b40-4f87-a8bc-52da8e758680)

**Comment by awcodes** [comment]
Probably because you're not including the views in a custom theme. It's all covered in the installation steps found on the Readme.

**Reply by nadeaumarc** [reply]
I did include the views, maybe I got something wrong?

![image](https://github.com/awcodes/filament-table-repeater/assets/127355317/d9501a34-354f-4f08-87ad-523513e2269c)
![image](https://github.com/awcodes/filament-table-repeater/assets/127355317/f5bee424-31b2-4bd5-9f6a-45f5c667605c)

**Reply by awcodes** [reply]
that looks ok to me. did you rerun the build?

**Reply by awcodes** [reply, marked as answer]
![Screenshot 2024-03-08 at 9 00 07 AM](https://github.com/awcodes/filament-table-repeater/assets/3596800/19c2496e-4ebf-47e0-a7a0-076e0aa97797)

**Reply by nadeaumarc** [reply]
Yes I did, turns out the path in vite.config.js was wrong, I had 'resources/css/filament/admin/theme.css' instead of 'resources/css/filament/app/theme.css'. 

Thank you for your help, got me on the right track! Appreciate it and great work on the plugins by the way!
